src/qudi/gui/scanning/scannergui.py

```diff
@@ -439,6 +418,22 @@ def _init_static_dockwidgets(self):
         self.tilt_correction_dockwidget.visibilityChanged.connect(self._mw.action_view_tilt_correction.setChecked)
         self._mw.action_view_tilt_correction.triggered[bool].connect(self.tilt_correction_dockwidget.setVisible)
 
+    def _set_optimizer_dockwidget(self):
```

Comment:
Ambigous name of function; it's not a setter. Maybe _init_optimizer_dockwidget()?

Reply:
changed the name